src/xcode/ENA/ENA/Source/Services/DownloadedPackagesStore/__tests__/DownloadedPackagesSQLLiteStoreTests.swift

```diff
@@ -413,15 +422,15 @@ final class DownloadedPackagesSQLLiteStoreTests: XCTestCase {
 		XCTAssertEqual(store.allDays(country: "DE").count, 2)
 
 		// add some more data
-		try store.set(country: "DE", day: "2020-06-03", etag: revokedEtag, package: package)
+		try? store.set(country: "DE", day: "2020-06-03", etag: revokedEtag, package: package)
```

Comment:
why optional? if it fails, the test function (re-)throws the error. No need to omit a potential issue here!
```suggestion
		try store.set(country: "DE", day: "2020-06-03", etag: revokedEtag, package: package)
```

Reply:
Oh and IF this throws we can test for that case with `XCTAssertThrow`.

Reply:
It will definitely throw because with this change we throw an error when the package is on the revoke list.
Don't need to test for the throw. Only need to test, wether the package was persisted or not.

Reply:
We expect the error to be thrown, this SHOULD be tested. The test will move on after the assert.

Reply:
done